go/pkg/casng/BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "casng",
```

Comment:
what will it do? / what casng stands for?

Reply:
This package will eventually include a full upload and download implementations for the CAS API. I wanted to reuse the name `cas`, but that will cause headaches with conflicts with the existing code in that package. The best short name I could come up with is `casng` for CAS next-gen 🤷